src/Register.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import {validateAll} from 'indicative/validator';
+
+export class Register extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            login: '',
+            email: '',
+            password: '',
+            password_confirmation: '',
+            errors: {}
```

Comment:
I suggest not to mix up filed values and additional data like errors,
Might be better to make the next structure for the state:
```
this.state = {
  form: { ... }
  errors: {}
```